fixtures/modal.fixture.js

```diff
@@ -0,0 +1,43 @@
+import {ModalLauncher, StandardModal} from "@khanacademy/wonder-blocks-modal";
+import Button from "@khanacademy/wonder-blocks-button";
+import {Body} from "@khanacademy/wonder-blocks-typography";
+
+import {describe, expect, test, render, simulate, sleep} from "./helpers.js";
+
+describe("Modal", () => {
+    test("StandardModal", async () => {
+        const standardModal = ({closeModal}: any) => (
+            <StandardModal
+                title="Title"
+                subtitle="You're reading the subtitle!"
+                content={
+                    <Body tag="p">
+                        {
+                            "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est."
+                        }
+                    </Body>
+                }
+                footer={
+                    // TODO(mdr): Use Wonder Blocks Button.
+                    <Button onClick={closeModal}>Close modal</Button>
+                }
+            />
+        );
+
+        await render(
+            <ModalLauncher modal={standardModal}>
+                {({openModal}) => (
+                    <Button onClick={openModal}>Standard modal</Button>
+                )}
+            </ModalLauncher>,
+        );
+        await simulate({type: "mousemove", clientX: 50, clientY: 25});
+        await simulate({type: "click"});
+        await sleep(100);
+        await expect(document.body).toMatchScreenshot();
+        await simulate({type: "keypress", key: "tab"});
+        await expect(document.body).toMatchScreenshot();
+        await simulate({type: "keypress", key: "tab"});
+        await expect(document.body).toMatchScreenshot();
```

Comment:
@matchu checkout the modal screenshots below.  They show that focus returns to the button that had focus when the modal was first opened.